Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -198,6 +198,9 @@ YUI.add('ez-platformuiapp', function (Y) {
                     oldService.setNextViewServiceParameters(newService);
                 }
             });
+
+            // Saving default value of scrollToTop for searchview to use.
+            this.set('scrollToTopDefault', this.get('scrollToTop'));
```

Comment:
I believe you should do it differently. You should set a default value in the attribute (in `ATTRS`) with read-only permission:
```javascript
scrollToTopDefault: {
  value: true,
  readOnly: true,
}
```
Then this line, you added, will be not needed.